joint_teapot/workers/gitea.py

```diff
@@ -401,8 +396,29 @@ def get_all_teams(
                     f"Failed to get members of team {team_id} in {self.org_name}: {e}"
                 )
                 continue
-            ret.append({"name": team_raw.name, "members": members})
-        return ret
+            res[team.name] = members
+        return res
+
+
+    def unsubscribe_from_repos(self, pattern: str) -> None:
+        regex = re.compile(pattern)
+        subscriptions = [
+            sub
+            for sub in self.user_api.user_current_list_subscriptions()
+            if sub.owner.login == self.org_name
+            and re.search(regex, sub.name) is not None
+        ]
+        if len(subscriptions) == 0:
+            logger.warning(f"No subscribed repo matches the pattern {pattern}")
+        else:
```

Comment:
just `return`

Reply:
Can you remind me again where this was? I did a force push and the line number changed.

Reply:
done